common/layout/layout-en.html

```diff
@@ -0,0 +1,758 @@
+		&lt;p&gt;This is the second column of the row.&lt;/p&gt;
+		&lt;p&gt;The row class provides a negative margin on the left and right sides to counteract the padding applied to the columns.&lt;/p&gt;
+	&lt;/div&gt;
+	&lt;/div&gt;
+	&lt;details&gt;
+	&lt;summary&gt;Page Details Summary&lt;/summary&gt;
+	&lt;p&gt;This is a details element inside a div with the <code>.pagedetails</code> class.&lt;/p&gt;
+	&lt;p&gt;It has padding of 2em at the top and bottom.&lt;/p&gt;
+	&lt;p&gt;The <code>.row</code> class is used for backwards compatibility with GCWeb 4.0.26 and below.&lt;/p&gt;
+	&lt;p&gt;The <code>details</code> element inside the <code>.row</code> class has margins applied.&lt;/p&gt;
+	&lt;p&gt;The first child of the <code>.row</code> class has a top margin, affecting the "Report a Problem" link.&lt;/p&gt;
+	&lt;/details&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>GCWeb override - Site footer <code>.row</code></h4>
+<div class="well">
+<footer class="gcweb-override-site-footer">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-4">
+				<h5>About Us</h5>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed eget est eu libero fringilla accumsan.</p>
+			</div>
+			<div class="col-md-4">
+				<h5>Contact Us</h5>
+				<p>Email: info@example.com<br>Phone: 123-456-7890</p>
+			</div>
+			<div class="col-md-4">
+				<h5>Follow Us</h5>
+				<ul class="list-inline">
+					<li class="list-inline-item"><a href="#" class="social-icon"><i class="fab fa-facebook"></i></a></li>
+					<li class="list-inline-item"><a href="#" class="social-icon"><i class="fab fa-twitter"></i></a></li>
+					<li class="list-inline-item"><a href="#" class="social-icon"><i class="fab fa-instagram"></i></a></li>
+				</ul>
+			</div>
+		</div>
+	</div>
+</footer>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well" &gt;&lt;footer class="gcweb-override-site-footer"&gt;
+	&lt;div class="container"&gt;
+		&lt;div class="row"&gt;
+			&lt;div class="col-md-4"&gt;
+				&lt;h5&gt;About Us&lt;/h5&gt;
+				&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed eget est eu libero fringilla accumsan.&lt;/p&gt;
+			&lt;/div&gt;
+			&lt;div class="col-md-4"&gt;
+				&lt;h5&gt;Contact Us&lt;/h5&gt;
+				&lt;p&gt;Email: info@example.com&lt;br&gt;Phone: 123-456-7890&lt;/p&gt;
+			&lt;/div&gt;
+			&lt;div class="col-md-4"&gt;
+				&lt;h5&gt;Follow Us&lt;/h5&gt;
+				&lt;ul class="list-inline"&gt;
+					&lt;li class="list-inline-item"&gt;&lt;a href="#" class="social-icon"&gt;&lt;i class="fab fa-facebook"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;
+					&lt;li class="list-inline-item"&gt;&lt;a href="#" class="social-icon"&gt;&lt;i class="fab fa-twitter"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;
+					&lt;li class="list-inline-item"&gt;&lt;a href="#" class="social-icon"&gt;&lt;i class="fab fa-instagram"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;
+				&lt;/ul&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/footer&gt;&lt;/div&gt;</code></pre>
+
```

Comment:
Section from Line 224 to 272 are needed to be deleted.
A new file site_footer-en.html & site_footer-fr.html have been created for the specified section